src/Symfony/Component/Yaml/Inline.php

```diff
@@ -584,8 +590,8 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         return log(0);
                     case 0 === strpos($scalar, '!!binary '):
                         return self::evaluateBinaryScalar(substr($scalar, 9));
-                    case preg_match('/^(-|\+)?[0-9,]+(\.[0-9]+)?$/', $scalar):
-                        return (float) str_replace(',', '', $scalar);
+                    case preg_match('/^(-|\+)?[0-9][0-9,_]*(\.[0-9_]+)?$/', $scalar):
```

Comment:
If you're referring to float, isn't it `1_200.2_200` ? Looking at http://yaml.org/type/float.html, in the regexp there is no `,` in fact (while there is a `_`)